src/components/camera/camera.test.jsx

```diff
@@ -0,0 +1,12 @@
+import { render } from "@testing-library/react";
+import { Camera } from "./Camera";
```

Comment:
This should be a lowercase `./camera`

I think this is causing that test to fail; please fix before merging